PWGJE/EMCALJetTasks/AliAnalysisTaskJetChargeFlavourTemplates.cxx

```diff
@@ -44,7 +44,6 @@ AliAnalysisTaskJetChargeFlavourTemplates::AliAnalysisTaskJetChargeFlavourTemplat
   fCentMax(10),
   fJetRadius(0),
   JetChargeK(0.5),
-  MotherFraction(0.8),

```

Comment:
Even if you're not going to set it via your add task, you need to initialize `ProgenetorFraction` in your constructor